### ISSUE TITLE
Would you have soon a new CRAN release?

### ISSUE THREAD

**Issue by AlexPars** [Appsilon/shiny.semantic#152]
While the current CRAN version is already quite good, it will be quite beneficial to the visualization and data science community to use instead the version containing all the enhancements that were done since 2018 on both shiny.semantic and SemanticUI

**Comment by dokato** [comment]
@AlexPars I'm glad you're asking. Final polishing and we should release 0.3.0 version o CRAN in a week or two ;)

**Comment by AlexPars** [comment]
This is very good news.

Would your team also have a new CRAN release for semantic.dashboard as well?

**Comment by AlexPars** [comment]
I wanted to inquire whether you have any updated timeline for CRAN release of the new version of this R package

**Comment by dokato** [comment]
Hey @AlexPars we're really sorry that we keep you waiting. We just had some holiday season and there's still a handful of polishing touches that need to be done before release. But please bear with us, it should be out there really soon.

**Comment by AlexPars** [comment]
No worries, and thank you for the update.

**Comment by dokato** [comment]
@AlexPars version 0.3.0 of `shiny.semantic` is merged into master now and submitted to CRAN. It usually takes around 10d for it to appear. Unless we get some extra comments that we would need to address.

**Comment by dokato** [comment]
@AlexPars actually that went fast. Version 0.3.0 is out on CRAN: https://cran.r-project.org/web/packages/shiny.semantic/index.html . New dashboard is pending...

**Comment by AlexPars** [comment]
Thank you for the hard work towards releasing version 0.3.0 on CRAN. Looking forward to upcoming CRAN release for semantic.dashboard
